src/main/java/picard/illumina/parser/fakers/MultiTileBclFileFaker.java

```diff
@@ -29,4 +38,25 @@ protected boolean addLeadingZeros() {
     protected int bufferSize() {
         return ((size - Integer.SIZE) * tiles.size()) + Integer.SIZE;
     }
-}
+
+    public void fakeMultiTileBclFile(final File bcl, final TileIndex tileIndex) throws IOException {
+        this.tiles = tileIndex.getTiles();
+        final OutputStream outputStream;
+        if (BclReader.isGzipped(bcl)) outputStream = new GZIPOutputStream(new FileOutputStream(bcl));

```

Comment:
I think the picard standard is to not do single line if statements:
```suggestion
        if (BclReader.isGzipped(bcl)) {
            outputStream = new GZIPOutputStream(new FileOutputStream(bcl));
        }...

```